src/codeu/chat/ServerMain.java

```diff
@@ -64,6 +65,10 @@ public static void main(String[] args) {
       final DatabaseAccess database = new DatabaseAccess();
       database.initialize();
       LOG.info("FireBase intialized");
+      // load users
+      final DatabaseStartup dbStartup = new DatabaseStartup();
+      dbStartup.getUsers();
+      dbStartup.getConversations();
```

Comment:
perhaps do this in the model constructor. That way you can directly reload the data into the model when it is created. You can even use those load methods that you created in the model